pkg/controller/utils/auth.go

```diff
@@ -75,6 +75,7 @@ func GetKeyValidatorConfig(ctx context.Context, cli client.Client, authenticatio
 
 // GetIdpSecret retrieves the Secret containing sensitive information for the configuration IdP specified in the given
 // operatorv1.Authentication CR.
+// TODO: This needs to be updated to support multiple tenants IDP secrets.
```

Comment:
The authentication spec will likely not be MT. Users log in through the cloud portal (Auth0). The idp secret is then the secret for the cloud portal in this case, and so it won't be MT either in that case.